app/src/main/java/com/davidtomas/taskyapp/features/agenda/presentation/agenda/AgendaViewModel.kt

```diff
@@ -106,12 +114,24 @@ class AgendaViewModel(
                 viewModelScope.launch {
                     authRepository.logout().fold(
                         onError = {},
-                        onSuccess = {}
+                        onSuccess = {
+                            _uiEvent.send(AgendaUiEvent.NavigateToLogin)
+                        }
                     )
                 }
             }
 
             else -> Unit
         }
     }
+
+    /*private fun scheduleSyncWorker() {
+        val workRequest = PeriodicWorkRequestBuilder<SyncDataWorker>(2, TimeUnit.MINUTES)
+            .build()
+        WorkManager.getInstance(applicationContext).enqueueUniquePeriodicWork(
+            "sync_data_work",
+            ExistingPeriodicWorkPolicy.CANCEL_AND_REENQUEUE,
+            workRequest
+        )
+    }*/
```

Comment:
IMO, it's a code smell to comment out code AND commit that, unless there's a really good reason for it (which there rarely is in my experience).
I've seen codebases where such commented out code stays in forever because one developer thought at some point that it's not needed right now, but might in future. And that'll be the same thing other developers will think ( `Oh, let's better not remove this commented out code, as it might be needed in future`).
All the code you're commenting out is already in version control (and if it's not - it was never important), so you'll find it there if you need it back. 

That of course only refers to committing such comments. If you comment out a section just to quickly try out something, that's totally fine.